modules/exploits/windows/local/dnsadmin_serverlevelplugindll.rb

```diff
@@ -0,0 +1,184 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+##
+
+class MetasploitModule < Msf::Exploit::Local
+  Rank = GreatRanking
```

Comment:
I'm not sure great applies here since service reboots are required, and the service will be unresponsive until after cleanup

Reply:
In that case, I think "Normal" fits better, but please advise. Changed to "NormalRanking" as of commit 78d87a0fced327b3de359a8946c387669cbfcb03.